src/main/java/com/dnd/namuiwiki/domain/survey/SurveyService.java

```diff
@@ -62,4 +70,47 @@ private User getUserByAccessToken(String accessToken) {
                 .orElseThrow(() -> new ApplicationErrorException(ApplicationErrorType.NOT_FOUND_USER));
     }
 
+    public GetAnswersByQuestionResponse getAnswersByQuestion(String wikiId, String questionId, Period period, Relation relation, int pageNo, int pageSize) {
+        validateFilter(period, relation);
+
+        Question question = questionRepository.findById(questionId)
+                .orElseThrow(() -> new ApplicationErrorException(ApplicationErrorType.INVALID_QUESTION_ID));
+
+        User owner = userRepository.findByWikiId(wikiId)
+                .orElseThrow(() -> new ApplicationErrorException(ApplicationErrorType.NOT_FOUND_USER));
+
+        Pageable pageable = PageRequest.of(pageNo, pageSize);
+        Page<Survey> surveys = getSurveysByFilter(period, relation, owner, pageable);
+        var answers = surveys.get().map(survey -> {
+                    var answerOfQuestion = survey.getAnswers().stream()
+                            .filter(answer -> answer.getQuestion().getId().equals(questionId))
+                            .findAny()
+                            .orElseThrow(() -> new ApplicationErrorException(ApplicationErrorType.INVALID_QUESTION_ID));
+                    return SingleAnswerWithSurveyDetailDto.builder()
+                            .senderName(survey.getSenderName())
+                            .period(survey.getPeriod())
+                            .relation(survey.getRelation())
+                            .answer(answerOfQuestion.getAnswer())
+                            .reason(answerOfQuestion.getReason())
+                            .build();
+        }).toList();
+
+        return new GetAnswersByQuestionResponse(question.getTitle(), answers);
+    }
+
+    private void validateFilter(Period period, Relation relation) {
+        if (!period.equals(Period.TOTAL) && !relation.equals(Relation.TOTAL)) {
+            throw new ApplicationErrorException(ApplicationErrorType.INVALID_FILTER);
+        }
+    }
+
+    private Page<Survey> getSurveysByFilter(Period period, Relation relation, User owner, Pageable pageable) {
+        if (!period.equals(Period.TOTAL)) {
```

Comment:
```suggestion
        if (!period.isTotal()) {
```
이 함수를 추가하고 메소드를 호출하는게 어떨까요?

```java
    public boolean isTotal() {
        return this == TOTAL;
    }
```